git/02.development-workflow.md

```diff
@@ -18,6 +18,10 @@ supercede general best practices.
 Start by creating a branch from the [release branch](07.release-branching.md)
 and switching to it.
 
+```
+$ git checkout -b feature_branch_name
+```
```

Comment:
Part of the reason for having the step by step instructions is we often point Academy or Junior people as recipe while they learn the why behind the approach.